network.go

```diff
@@ -1938,6 +1938,22 @@ func (n *network) TableEventRegister(tableName string, objType driverapi.ObjectT
 	return nil
 }
 
+func (n *network) UpdateIpamConfig(ipV4Data []driverapi.IPAMData) {
```

Comment:
I still stick by my comment. if we are not using it, we dont need it.  It will confuse everyone who is looking into the code after few months wondering why is this code exists if we are not using it.